website/docs/llm_configuration.ipynb

```diff
@@ -0,0 +1,350 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "custom_edit_url: https://github.com/microsoft/autogen/edit/main/website/docs/llm_configuration.ipynb\n",
+    "---\n",
+    "\n",
+    "# LLM Configuration\n",
+    "\n",
+    "In AutoGen, LLM's are a key component required for agents to understand and react. Each agent's access to and configuration of LLM's is defined by the `llm_config` argument in its constructor.\n",
+    "\n",
+    "The simplest possible LLM configuration that is configured to use `gpt-4` from OpenAI would be:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "llm_config = (\n",
+    "    {\n",
+    "        \"config_list\": [{\"model\": \"gpt-4\", \"api_key\": os.environ[\"OPENAI_API_KEY\"]}],\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "````{=mdx}\n",
+    ":::warning\n",
+    "It is important to never commit secrets into your code, therefore we read the OpenAI API key from an environment variable.\n",
+    ":::\n",
+    "````\n",
+    "\n",
+    "This `llm_config` can then be passed to an agent's constructor to enable it to use the LLM."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import autogen  # noqa: E402\n",
+    "\n",
+    "assistant = autogen.AssistantAgent(name=\"assistant\", llm_config=llm_config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## `config_list`\n",
```

Comment:
Maybe add English around this heading. It looks weird in the pdf you shared and also too big.

Reply:
Yeah happy to add that. It looks better on the website but I still think that little bit of extra wording is nice